src/clp_logging/handlers.py

```diff
@@ -52,9 +52,11 @@ def _init_timeinfo(fmt: Optional[str], tz: Optional[str]) -> Tuple[str, str]:
     if not tz:
         tzf = dateutil.tz.gettz()
         if tzf:
-            if tzf._filename == "/etc/localtime":
-                tzp = Path.resolve(Path(tzf._filename))
+            if tzf._filename == "/etc/localtime":  # type: ignore
```

Comment:
I changed this code to always resolve the tsf._filename path rather than only if it is set to local time (as afaik there could be other aliasing not accounted for).